admin.php

```diff
@@ -290,6 +290,9 @@ public function onPluginsInitialized()
             };
         }
 
+        // Load utility class
+        require_once __DIR__ . '/classes/utils.php';
```

Comment:
Sorry, misread - it's a new file. But it triggers the problem.. do we really need a separate file just for `findUserbyEmail`?